__tests__/test-statusMatrix.js

```diff
@@ -394,4 +394,109 @@ describe('statusMatrix', () => {
       ['b/b.txt', 0, 2, 2],
     ])
   })
+
+  describe('statusMatrix with a symlinked directory', () => {
+    const realFolder = 'realFolder'
+    let storedProps = {}
+    const symlinkCreated = false
+    beforeAll(async () => {
```

Comment:
Please rewrite the tests and don't rely on sequence, the tests can be run in parallel and it will break the test if one case depends on another case. Use `beforeEach` and prepare new base repo for all tests.